src/Wallet/KeyFile.purs

```diff
@@ -3,10 +3,15 @@ module Wallet.KeyFile
   ( privatePaymentKeyFromFile
   , privateStakeKeyFromFile
   , keyFromFile
+  , privatePaymentKeyToFile
```

Comment:
Please re-export `formatStakeKey` and `formatPaymentKey` from `Contract.Wallet` and `privatePaymentKeyToFile` and `privateStakeKeyToFile` from `Contract.Wallet.KeyFile`.